ccd-definition/AuthorisationCaseField/cafcass.json

```diff
@@ -294,6 +294,13 @@
     "UserRole": "caseworker-publiclaw-cafcass",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "c2ApplicationType",
+    "UserRole": "caseworker-publiclaw-cafcass",
+    "CRUD": "CRU"
```

Comment:
Are they going to need the `D` 😉

Reply:
copy-pasted existing lasolicitor permissions, but after checking it looks like it's unnecessary, so removing